pkg/provider/provider.go

```diff
@@ -184,7 +186,7 @@ func (mc *mountConfig) GetServicePrincipalToken(resource string) (*adal.ServiceP
 }
 
 // MountSecretsStoreObjectContent mounts content of the secrets store object to target path
-func (p *Provider) MountSecretsStoreObjectContent(ctx context.Context, attrib map[string]string, secrets map[string]string, targetPath string, defaultFilePermission os.FileMode) ([]SecretFile, map[string]string, error) {
+func (p *Provider) MountSecretsStoreObjectContent(ctx context.Context, attrib, secrets map[string]string, targetPath string, defaultFilePermission os.FileMode) ([]SecretFile, error) {
```

Comment:
Since we are not mounting secrets anymore in provider, should we change name of this function? May be to `FetchSecretsStoreObjectContent`

Reply:
Updated it to `GetSecretsStoreObjectContent`